test/users.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require('supertest'),
+  app = require('../app');
+
+describe('POST /user', () => {
+  it('signUp User', done => {
```

Comment:
the messages should be more descriptive: "should create a new user because all parameters are send and valid" for example

Reply:
ok